src/Controller/UserController.php

```diff
@@ -109,20 +109,19 @@ public function deleteUser(Request $request, array $requestAttributes): Response
         $this->userService->delete($id);
         $body = Stream::createFromString('');
         $response = new Response($body, '1.1', 301, '');
-        $response = $response->withHeader('Location', 'http://local.quiz.com/listUsers');
+        $response = $response->withHeader('Location', 'http://local.quiz.com/listUsers?page=1');
 
         return $response;
     }
 
     public function getUsers(Request $request, array $requestAttributes): Response
     {
```

Comment:
What happens if the page is accessed without the page parameter? Will it break with an error?